packages/vector_graphics_compiler/lib/src/svg/parser.dart

```diff
@@ -686,24 +684,6 @@ class SvgParser {
     }
   }
 
-  bool _isShapeOrText(String elementName) {
```

Comment:
Not needed anymore because resolution of this happens after parsing is done in the resolver now, which knows whether it's dealing with a shape or a group.